app/views/entities/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="new-category-nav">
+<%= render "shared/back" %>
+<h3 class="new-category-title">Transactions</h3>
+</div>
+<main class="transactions-container">
+    <% if @entities.empty? %>
+    <div class="empty-category">
+        <h3>You don't have any transactions yet! <%= @category.name %> Category</h3>
+        <%= link_to 'Add a new transaction', new_category_entity_path %>
+    </div>
+<% else %>
+    <div class="total-transactions">
+        <span>
+            <%= @category.name %>
+        </span>
+        <Span>
+            $<%= @entities.inject(0) { |sum, e| sum += e.amount  }%> 
+        </Span>
+    </div>
+    <div class="transactions">
+        <% @entities.each do |entity| %>
+            <div class="transaction">
+                    <%= image_tag(@category.icon || '', class: "category-img") %>
+                <div>
+                    <h2>
+                        <%= entity.name %>
+                    </h2>
+                    <span>
+                        <%= entity.created_at.strftime("%d %b %Y") %>
+                    </span>
+                </div>
+                <div>
+                    <Span>
+                        $<%= entity.amount %> 
+                    </Span>
+                </div>
+
+                    </div>
+                    <% end %>
```

Comment:
- [ ] Kindly consider rendering only transactions which added to the specific category because now if I have category 1 and  2 and If I add two transactions in category 1, It will display in category 2 although it was empty

Reply:
done

Reply:
- [ ] As the previous reviewer mentioned, when I add a transaction to a second category, the first category takes the amount, kindly fix this, the below picture is after I added a transaction to the pot category but  the amount increased in the Game category
![Screenshot from 2023-05-20 06-37-57](https://github.com/bill7pearl/BudgetWise/assets/66097371/b0ef2fb5-e529-4985-8130-de74a3383e58)